Core/include/Acts/Surfaces/detail/VerticesHelper.hpp

```diff
@@ -234,38 +151,20 @@ bool isInsidePolygon(const vertex_t& point,
 ///                 an `Vector2DType`.
 /// @return bool for inside/outside
 template <typename vertex_t>
-bool isInsideRectangle(const vertex_t& point, const vertex_t& lowerLeft,
-                       const vertex_t& upperRight) {
+inline bool isInsideRectangle(const vertex_t& point, const vertex_t& lowerLeft,
+                              const vertex_t& upperRight) {
   return (lowerLeft[0] <= point[0]) && (point[0] < upperRight[0]) &&
          (lowerLeft[1] <= point[1]) && (point[1] < upperRight[1]);
 }
 
-/// This method checks if a cloud of points are on 2D hyper-plane in 3D space
+/// This method checks if a cloud of points are on 2D hyper-plane in 3D space.
 ///
 /// @param vertices The list of vertices to test
 /// @param tolerance The allowed out of plane tolerance
-///
-/// It bails out at the first moment a point is outside the hyper plane
 /// @return boolean to indicate if all points are inside/outside
-static bool onHyperPlane(const std::vector<Vector3D>& vertices,
-                         double tolerance = s_onSurfaceTolerance) {
-  // Obvious always on one surface
-  if (vertices.size() < 4) {
-    return true;
-  }
-  // Create the hyperplane
-  auto hyperPlane = Eigen::Hyperplane<double, 3>::Through(
-      vertices[0], vertices[1], vertices[2]);
-  for (size_t ip = 3; ip < vertices.size(); ++ip) {
-    if (hyperPlane.absDistance(vertices[ip]) > tolerance) {
-      return false;
-    }
-  }
-  return true;
-}
-
-};  // namespace VerticesHelper
+bool onHyperPlane(const std::vector<Vector3D>& vertices,
+                  double tolerance = s_onSurfaceTolerance);
 
+}  // namespace VerticesHelper
 }  // namespace detail
-
-}  // namespace Acts
\ No newline at end of file
+}  // namespace Acts
```

Comment:
Maybe I'm the only one that cares about this, but I always try to keep the newline changes at the end of files out of the diff.

Reply:
Removed.